core/tests/protractor_utils/editor.js

```diff
@@ -139,9 +139,25 @@ var expectContentTextToEqual = function(text) {
 
 // INTERACTIONS
 
+// Used as the standard way to specify interaction for most purposes.
 // Additional arguments may be sent to this function, and they will be
 // passed on to the relevant interaction editor.
 var setInteraction = function(interactionId) {
+  openInteraction(interactionId);
+  customizeInteraction.apply(null, arguments);
+  // If the "Add Response" modal opens, close it.
+  var headerElem = element(by.css(
+    '.protractor-test-add-response-modal-header'));
+  headerElem.isPresent().then(function(isVisible) {
+    if (isVisible) {
+      closeAddResponseModal();
+    }
+  });
+};
+
+// This function is not usually necessary, please consider using
```

Comment:
This function should not usually be invoked directly; please consider using setInteraction instead.